src/platforms/javascript/common/session-replay/index.mdx

```diff
@@ -31,6 +31,10 @@ By default, it'll mask all fields, so no personally identifiable information (PI
 
 <PlatformContent includePath="session-replay/pre-requisites" />
 
+<Alert level="warning">
+  We’re investigating possible performance degradation in some **Angular applications**. We recommend testing your application before releasing. Please reach out via [this GitHub ticket](https://github.com/getsentry/sentry-javascript/issues/6946) if you have issues.
+</Alert>
+
```

Comment:
```suggestion
<Alert level="warning">

 We’re investigating possible performance degradation in some **Angular applications**. We recommend testing your application before releasing. Please reach out via <a class href=“https://github.com/getsentry/sentry-javascript/issues/6946”>this GitHub ticket</a> if you have issues.
 
</Alert>

```
Links haven't been working in alerts/notes for some reason. ^^ they do seems to work if we write them this way.

Reply:
Ah looks like markdown is not supported inside of react components, thanks!